bitnami/postgresql/templates/statefulset.yaml

```diff
@@ -76,6 +76,11 @@ spec:
             - -cx
             - |
               {{- if .Values.persistence.enabled }}
+              {{- if eq ( toString ( .Values.volumePermissions.securityContext.runAsUser )) "auto" }}
+              chown -R `id -u`:`id -G | cut -d " " -f2` {{ .Values.persistence.mountPath }}
+              {{- else }}
+              chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} {{ .Values.persistence.mountPath }}
```

Comment:
Also, remember to apply this changes to `statefulset-slaves` as well: https://github.com/bitnami/charts/blob/77f3cfe9f0c51871008c5a3b84bd7a95d93c939f/bitnami/postgresql/templates/statefulset-slaves.yaml#L78